### DISCUSSION TITLE
How to use Fastlane to enable capabilities in provision profile?

### DISCUSSION THREAD

**Discussion by OliverKoo** [fastlane/fastlane#19280] [Q&A]
Our org uses Fastlane to manage out codesign stuff. We have the github repo set up and been using it for a while.

For Xcode13 we added time sensitive usernotifications capability to our entitlement file.
```
  <key>com.apple.developer.usernotifications.time-sensitive</key>
  <true/>
```
From my research it seems like we have to re-generate the provision profile. How can I achieve this with Fastlane?

is using sigh the right way? I am hesitating to do any actions because all the iOS dev share the certificate git repo, so I am being very cautious.

I also made a StackOverflow post here - https://stackoverflow.com/questions/68913813/how-to-use-fastlane-to-enable-capabilities-in-provision-profile

**Comment by lucgrabowski** [comment]
@OliverKoo 
Before you regenerate the profile, you need to add a capability to app identifier. When you edit app identifier, all the profiles for this app will be invalid and you'll have to regenerate them. 
You can enable capabilities in Developer Center or with `fastlane`'s: 
`produce enable_services` (http://docs.fastlane.tools/actions/produce/#parameters_1)
or with `modify_services` action https://docs.fastlane.tools/actions/modify_services/
The name of the `service` that you want to enable is `time_sensitive_notifications`
